cub/cub/device/device_run_length_encode.cuh

```diff
@@ -224,6 +224,7 @@ struct DeviceRunLengthEncode
 
     using policy_t = detail::device_run_length_encode_policy_hub<accum_t, key_t>;
 
+
```

Comment:
nit: we can remove this superfluous empty line / or clang-format the file

Reply:
Thanks a lot